charts/istio-authorizer/values.yaml

```diff
@@ -83,6 +83,11 @@ parseBody:
   ##
   maxRequestBytes: 8192
 
+  ## Custom list of namespaces that the parse body filter will be applied to.
+  ## If empty the discovery namespaces will be used.
```

Comment:
I'm not sure what do we mean about the discovery namespaces here.
Maybe sth like: If empty, namespaces from the `data.config.discovery.namespace` parameter are used.